Tests/kaas/k8s-version-policy/testdata/README.md

```diff
@@ -0,0 +1,10 @@
+# Testdata for unit tests
+
+## releases.json
+
+was created from real release data filtered through `jq`:
```

Comment:
maybe add some note that this file can be left as-is and the command is just FYI
(otherwise one might think that this command is meant to be issued by the user)